README.md

```diff
@@ -80,20 +84,13 @@ After the 16 week course, students have a number of options:
 
 **We believe all of these can co-exist.**
 
-This proposes that a ***commercial-focussed organisation*** is formed
-- run by [@nelsonic] and [@iteles] in the immediate term -
+This proposes that a ***commercial-focussed organisation*** is formed - run by [@nelsonic] and [@iteles] in the immediate term -
```

Comment:
On the one hand, changing the name _now_ may raise a few new questions which distract from the core purpose of the proposal. On the other, they are intertwined, so I suggest we _do_ change the name now so that those questions can be addressed early on in the process.